presto-hive/src/main/java/com/facebook/presto/hive/HiveColumnHandle.java

```diff
@@ -33,14 +39,52 @@
         implements ColumnHandle
 {
     public static final String SAMPLE_WEIGHT_COLUMN_NAME = "__presto__sample_weight__";
+
+    public static final int PATH_COLUMN_INDEX = -11;
+    public static final String PATH_COLUMN_NAME = "$path";
+    public static final HiveType PATH_HIVE_TYPE = HIVE_STRING;
+    public static final TypeSignature PATH_TYPE_SIGNATURE = PATH_HIVE_TYPE.getTypeSignature(false);
+
     private static final String UPDATE_ROW_ID_COLUMN_NAME = "$shard_row_id";
 
+    public enum ColumnType
```

Comment:
Thanks! Fixed.

I handled serialization manually before because `TestJsonHiveHandles` will failed if I directly put the enum type `PARTITION_KEY` in the map (it seems that Jackson would not deserialize enum type until necessary). 

Now I could make it work by `PARTITION_KEY.toString()`.